src/adafruit_blinka/board/orangepi/orangepi3b.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-FileCopyrightText: 2021 Melissa LeBlanc-Williams for Adafruit Industries
```

Comment:
Can you update the copyright to your name or username, and the year to 2024. You can leave of the "for Adafruit Industries" as well. 

This is looking good to me no beyond that.

Reply:
done.